tests/jballerina-unit-test/src/test/resources/test-src/expressions/binaryoperations/multiply-operation-negative.bal

```diff
@@ -45,3 +45,9 @@ function testImplicitConversion() {
     var y1 = d * e;
     any y2 = d * e;
 }
+
+function overflowByMultiplication() {
+    int num1 = -1;
+    int num2 = -9223372036854775808;
+    int ans = num1 * num2;
```

Comment:
IMO this test will not check overflow of multiplication since already there is a compile error for `int num2 = -9223372036854775808;` and program will fail before the runtime (where the multiplication is happened). We can test multiplication of constants in compile time. BTW currently it gives incorrect results. 
Ex.
```ballerina
const num1 = -2;
const  num2 = -9223372036854775807;
const int ans = num1 * num2; // ans = -2
```
Might be needed to fix separately.

Reply:
Created #36648 to track